README.md

```diff
@@ -353,31 +353,24 @@ Navigate to [New Release Form](https://github.com/CDCgov/prime-simplereport/rele
 ![full-dialog](https://user-images.githubusercontent.com/28784751/121424756-b31bd380-c93f-11eb-987d-38934f0570ae.png)
 
 1. <img align="right" width="517" alt="select-commit" src="https://user-images.githubusercontent.com/28784751/121423065-df365500-c93d-11eb-9b95-a63130d602e6.png">
-   Select the commit you want to release. This is likely to be the last commit on `main`, but select
-   the commit explicitly so that you do not accidentally release changes that somebody else is in the
+   Select the last commit on `main`. Selecting
```

Comment:
Looks like the formatting got a little chopped up?

Reply:
this is intentional. We should no longer be cutting releases at anything other then the latest commit